static/defaults.config.php

```diff
@@ -59,6 +59,10 @@
 		// Only show small preview pictures.
 		'always_show_preview' => false,
 
+		// ap_always_bcc (Boolean)
+		// Adressing receivers via ActivityPub by BCC. Increases privacy, decreases performnce.
```

Comment:
`Adresses non-mentioned ActivityPub receivers by BCC instead of CC. Increases privacy, decreases performance.`